master/freebsd_ci.py

```diff
@@ -0,0 +1,227 @@
+###############################################################################
+#  Workers
+###############################################################################
+
+freebsdci_workers = [
+    bworker.Worker('abeing', 'pass', max_builds=1),
+    bworker.Worker('gaebolg', 'pass', max_builds=1),
+    bworker.Worker('csisw3', 'pass', max_builds=1),
+    bworker.Worker('fragarach', 'pass', max_builds=1),
+    bworker.Worker('caladbolg', 'pass', max_builds=1),
+    bworker.Worker('rhongomyniad', 'pass', max_builds=1),
+    bworker.Worker('hrunting', 'pass', max_builds=1),
+    bworker.Worker('balmung', 'pass', max_builds=1),
+]
```

Comment:
Wow thats a lot of workers. Let’s put this in inventory.py along with the rest of the buildbot inventory.

Reply:
Do you want to add some prefix or patterns about the naming of workers?

Reply:
Ideally we would have some system like `<os>-<arch>-<identifier>` which would make it easy for us to do things like add multiple workers to the same scheduler.  The buildbots have, so far, only had one worker per OS/Arch.  But we've only been running on commits that actually make it into `master`, we haven't been doing PRs and whatnot, so we may soon need more than one worker per OS/Arch.  So I'm imagining we might have a system such as:

```
linux-x86_64-worker1
linux-aarch64-yuyichao
macos-x86_64-elliot
freebsd-x86_64-balmung
freebsd-x86_64-hrunting
....
```

This way, we can have a more-or-less freeform identifier at the end for our human recognition, while still being able to do things like say the "linux64" buildbots are `[w for w in workers if w.startswith("linux-x86_64-")]`.

What do you think?

Reply:
`<os>-<arch>-<identifier>` is fine for me.
Seems there are a version number pattern in the `inventory.py`: `<os>-<ver>-<arch>`
I prefer dropping the version number.

Reply:
Yes, I strongly agree.  It was necessary back when we had multiple OSX buildbots (we had homebrew bottling buildbots that would build things for osx 10.7, 10.8, 10.9, etc....) but nowadays we have become wise enough to just build for the oldest possible. :)